redisinsight/ui/src/components/cli/components/cli-input/CliInputWrapper.tsx

```diff
@@ -6,6 +6,7 @@ import { appRedisCommandsSelector } from 'uiSrc/slices/app/redis-commands'
 import CliAutocomplete from './CliAutocomplete'
 
 import CliInput from './CliInput'
+import { outputSelector } from 'uiSrc/slices/cli/cli-output';
```

Comment:
Could you move this absolute path import before relative imports?

Reply:
done